src/libraries/Microsoft.PowerFx.Core/Public/Types/AggregateType.cs

```diff
@@ -58,6 +58,26 @@ public virtual bool TryGetFieldType(string name, out FormulaType type)
             return true;
         }
 
+        internal bool TryGetBackingDType(string name, out DType type)
```

Comment:
>name [](http://example.com/codeflow?start=48&length=4)

suggestion: fieldName (as is defined in GetFieldType). The function above (TryGetFieldType) is explicit named about fields, this one is not.

Reply:
updated